sharding/chainParametersHolder.go

```diff
@@ -68,6 +75,18 @@ func validateArgs(args ArgsChainParametersHolder) error {
 		}
 	}
 
+	doesConfigForEpochZeroExist := false
+	for _, chainParams := range chainParametersConfig {
```

Comment:
this for can be avoided and add a check on L36, just after you sort the configs (same as in ratingsData.go)